examples/async-chat-stream/chat.ts

```diff
@@ -0,0 +1,102 @@
+import { spawn, ChildProcessWithoutNullStreams, execSync } from 'child_process'
```

Comment:
Each of these should be a few lines of code – copy and paste-able

Reply:
removed this example